frontend/pages/register/index.tsx

```diff
@@ -1,3 +1,12 @@
+import Link from "next/link"
+
 export default function Register() {
-  return <div className="Register">I AM REGISTRATION!!!!!!</div>
+  return (
+    <>
+      <div className="Register">I AM REGISTRATION!!!!!!</div>
+      <Link href="/register/passport">
```

Comment:
I am pretty sure the flow is always the user must register as a basic user then from there once they are logged in they can then request to update to a passport user or behyond.

Reply:
This comment may be disregarded since i see the PR for basic user registration so maybe just need to link this one from the dashboard?

Reply:
It's just a placeholder for me to test that routing was working as expected, it certainly doesn't need to stay here.